bezier-rs/docs/interactive-docs/src/App.vue

```diff
@@ -146,7 +148,48 @@ export default defineComponent({
 						drawPoint(context, normalEnd, 3, "Red");
 					},
 					template: markRaw(SliderExample),
-					templateOptions: tSliderOptions,
+					templateOptions: { sliders: [{ ...tSliderOptions }] },
+				},
+				{
+					id: 6,
+					name: "Split",
+					callback: (canvas: HTMLCanvasElement, bezier: WasmBezierInstance, options: Record<string, number>): void => {
+						const context = getContextFromCanvas(canvas);
+						const bezierPair = bezier.split(options.t);
+
+						drawBezierHelper(context, bezierPair.second, "orange", 3.5);
```

Comment:
"orange" is inconsistently not capitalized, but these are just CSS variables, right? I recommend making them all lowercase.